js/huobi.js

```diff
@@ -5026,17 +5095,40 @@ module.exports = class huobi extends Exchange {
         const currency = this.currency (code);
         const request = {
             'address': address, // only supports existing addresses in your withdraw address list
-            'amount': amount,
             'currency': currency['id'].toLowerCase (),
         };
         if (tag !== undefined) {
             request['addr-tag'] = tag; // only for XRP?
         }
-        const [ networkCode, paramsOmited ] = this.handleNetworkCodeAndParams (params);
+        let networkCode = undefined;
+        [ networkCode, params ] = this.handleNetworkCodeAndParams (params);
         if (networkCode !== undefined) {
-            request['chain'] = this.networkCodeToId (code, networkCode);
-        }
-        const response = await this.spotPrivatePostV1DwWithdrawApiCreate (this.extend (request, paramsOmited));
+            request['chain'] = this.networkCodeToId (networkCode, code);
+        }
+        amount = parseFloat (this.currencyToPrecision (code, amount, networkCode));
```

Comment:
@ttodua can we replace all `parseFloat` occurrences with `parseNumber` ? `parseFloat` does not take into consideration the precision mode (float or string) chosen by the user